per/factories.py

```diff
@@ -105,12 +107,22 @@ class Meta:
         model = FormPrioritization
 
 
+class AppealFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Appeal
+
+    country = factory.SubFactory(CountryFactory)
+
+
 class OpsLearningFactory(factory.django.DjangoModelFactory):
     learning = fuzzy.FuzzyText(length=50)
 
     class Meta:
         model = OpsLearning
 
+    appeal_code = factory.SubFactory(AppealFactory)
+    is_validated = fuzzy.FuzzyChoice([True, False])
```

Comment:
Let's not use fuzzy choice for is_validated, we should manually define in the test cases as lots of endpoints depends on this